tests/dummy_qrc/zurich.py

```diff
@@ -147,3 +180,455 @@ def create(runcard=RUNCARD):
         qubits[f"c{c}"].flux_coupler.append(qubits[2])
 
     return platform
+
+
+def test_random_functions():
```

Comment:
I think there is something wrong here. Test functions should be in the other file (test_instruments_zhinst.py).

Reply:
True